rpcs3/rpcs3qt/vfs_dialog_path_widget.cpp

```diff
@@ -123,13 +127,29 @@ void vfs_dialog_path_widget::add_new_directory()
 {
 	QString dir = QFileDialog::getExistingDirectory(nullptr, tr("Choose a directory"), QCoreApplication::applicationDirPath(), QFileDialog::DontResolveSymlinks);
 
+	// The dialog box always enables the focus but it doesn't restore the previuos state (e.g. "NoFocus") on closure.
+	// Any focus eventually present on the current row before opening the dialog box is also not restored.
+	// The code takes care and properly manages these possible changes
+
 	if (dir.isEmpty())
+	{
+		// Just to get the focus (otherwise lost by the dialog box opened by the "getExistingDirectory()" method)
+		m_dir_list->currentRowChanged(m_dir_list->currentRow());
```

Comment:
same issue as in the other comment

Reply:
see my other comment and  the comment above that code.